docs/event-dispatcher.md

```diff
@@ -100,10 +100,22 @@ Example:
       "amt": 5000
     }
   ],
+  "reward_slot_holders": [
+    "1C56LYirKa3PFXFsvhSESgDy2acEHVAEt6",
+    "1C56LYirKa3PFXFsvhSESgDy2acEHVAEt6"
+  ],
   "burn_amount": 12000
 }
 ```
 
+* `reward_recipients` is an array of all the rewards received during this burn block. It may
```

Comment:
I started working on similar in another branch a few weeks ago, and unless I missed something, if we want that list to be as exhaustive as possible, I think we'd need to tweak `LeaderBlockCommitOp::parse_from_tx` - ignoring a lot invalid block commit. I was thinking about producing some kind of `ConsensusReceipt` struct, that would help building a clear explanation of the consensus, and also returning the reasons for leader block commit being ignored.
Anyway this PR is way better than the approach I had in mind: it's live :).